src/Symfony/Bridge/Doctrine/Tests/ExpressionLanguage/DoctrineParserCacheTest.php

```diff
@@ -12,13 +12,25 @@
 namespace Symfony\Bridge\Doctrine\Tests\ExpressionLanguage;
 
 use Symfony\Bridge\Doctrine\ExpressionLanguage\DoctrineParserCache;
+use Symfony\Bridge\PhpUnit\ErrorAssert;
 
 class DoctrineParserCacheTest extends \PHPUnit_Framework_TestCase
```

Comment:
DoctrineParserCache should also be deprecated (and this whole test marked as legacy, no need for assertDeprecationsAreTriggered)